package.json

```diff
@@ -36,25 +36,30 @@
   "devDependencies": {
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.4.0",
+    "@babel/polyfill": "^7.4.3",
```

Comment:
What do you use @babel/polyfill for?

Reply:
Oh, good catch. That was one thing I was fiddling around with to fix some issues, but end up not using it. Will uninstall it!